extensions/git/src/commands.ts

```diff
@@ -254,7 +254,13 @@ export class CommandCenter {
 		}
 
 		const config = workspace.getConfiguration('git');
-		const value = config.get<string>('defaultCloneDirectory') || os.homedir();
+		let value;
+		const userInputtedDirectory = config.get<string>('defaultCloneDirectory');
+		if (userInputtedDirectory !== undefined && userInputtedDirectory.match(/^~/)) {
```

Comment:
Why not use `if (userInputtedDirectory && userInputtedDirectory.match(/^~/)) {`?

Reply:
@jcperez I could have written it that way and it would have worked, but I am being more explicit with the code I write these days.